### PR TITLE
Fixes wing thruster exploit

### PR DESCRIPTION
**Pull request by utilbird**
## About The Pull Request

Robos are currently able to implant wings and a thruster implant for quick and easy super speed 

## Why It's Good For The Game

powergame exploit bad

## Changelog
:cl:
fix: Thrusters now check for wings/flight
/:cl:

### CODE REVIEW

**Comment by eeSPee** [comment]
May be intended?

**Comment by Rukofamicom** [comment]
> reee that it's an intended feature

Some guy somewhere is thinking this.

**Comment by PowerfulBacon** [comment]
What if they are in fly mode in no grav

**Comment by utilbird** [comment]
good point

**Review by PowerfulBacon — APPROVED** [review]

**Comment by OceanFish1** [comment]
Feature


:trollface:

**Comment by PowerfulBacon** [comment]
why does FLOATING movespeed modifiers affect FLYING?

**Comment by KittyNoodle** [comment]
cringe pr

File: code/modules/surgery/organs/augments_chest.dm [hunk 1]
```diff
@@ -183,7 +183,8 @@
 	var/turf/T = get_turf(owner)
 	if(!T) // No more runtimes from being stuck in nullspace.
 		return 0
-
+	if(owner.is_flying() && owner.has_gravity())
+		return 0
```
Comment (KittyNoodle) [inline code/modules/surgery/organs/augments_chest.dm:187]: cringe

**Pull request merged**